src/Orleans/Streams/Internal/StreamSubscriptionHandleImpl.cs

```diff
@@ -39,7 +38,7 @@ public StreamSubscriptionHandleImpl(GuidId subscriptionId, StreamImpl<T> streamI
             this.observer = observer;
             this.streamImpl = streamImpl;
             this.filterWrapper = filterWrapper;
-            this.isRewindable = isRewindable;
+            this.isRewindable = streamImpl.IsRewindable;
             if (IsRewindable)
```

Comment:
let me know if you want me to remove Is(is)Rewindable member from StreamSubscriptionHandleImpl class, since I don't think it is used in other classes.